clang/lib/CConv/ProgramInfo.cpp

```diff
@@ -480,14 +480,17 @@ ProgramInfo::insertIntoExternalFunctionMap(ExternalFunctionMapType &Map,
     RetVal = true;
   } else {
     // MultipleRewrites = true;
+    auto oldS = Map[FuncName];
+    auto *newC = getOnly(ToIns);
+    auto *oldC = getOnly(oldS);
     if (isDef) {
```

Comment:
I don't think we need the `isDef` flag, since we can look in `newC` to see if `HaveBody()` is true, right?